### PR TITLE
Add basic support for bulk deleting all devices

### LINKED ISSUE #1435: 10.6.0 delete all devices
**Describe the bug**
web/index.html#!/devices.html
Dashboard > Devices
if i want to delete all devices, i have to do it one by one

**Expected behavior**
add a button "delete all devices"

**System (please complete the following information):**
 - OS: docker
 - Browser: Chrome
 - Jellyfin Version: 10.6.0

### PR DESCRIPTION
**Pull request by masterkoppa**
**Changes**
Make it easier to purge all devices from the device list by adding a single button action to do so.

Api calls are done in parallel, without error handling, but it should be a relatively fast and safe operation.

This required updating the help button size as well.

**Issues**
Fixes #1435

### CODE REVIEW

**Comment by masterkoppa** [comment]
I believe the issue raised by SonarCloud is a false positive, any advice on how to flag it or get SonarCloud to accept it would be appreciated.

https://sonarcloud.io/project/issues?id=jellyfin_jellyfin-web&issues=AXM_uu6VHQ9fbm-hecNo&open=AXM_uu6VHQ9fbm-hecNo&pullRequest=1552

File: src/controllers/dashboard/devices/devices.js [hunk 1]
```diff
@@ -1,10 +1,40 @@
 define(['loading', 'dom', 'libraryMenu', 'globalize', 'scripts/imagehelper', 'date-fns', 'dfnshelper', 'emby-button', 'emby-itemscontainer', 'cardStyle'], function (loading, dom, libraryMenu, globalize, imageHelper, datefns, dfnshelper) {
     'use strict';
+    // Local cache of loaded
+    let deviceIds = [];
 
     function canDelete(deviceId) {
         return deviceId !== ApiClient.deviceId();
     }
 
+    function deleteViaApi(id) {
```
Comment (MrTimscampi) [inline src/controllers/dashboard/devices/devices.js:10]: I would much rather have this moved to the API client itself.

Comment (masterkoppa) [reply src/controllers/dashboard/devices/devices.js:10]: Good call, will and link to this PR.

To be clear I'm assuming you mean adding it as a method off the `ApiClient` class [here](https://github.com/jellyfin/jellyfin-apiclient-javascript/blob/master/src/apiClient.js), right?

Comment (MrTimscampi) [reply src/controllers/dashboard/devices/devices.js:10]: Exactly :)

Comment (masterkoppa) [reply src/controllers/dashboard/devices/devices.js:10]: Done as part of jellyfin/jellyfin-apiclient-javascript#59

After the PR is merged, I'll update this PR to use the method and latest apiclient version in `package.json`

Comment (dkanada) [reply src/controllers/dashboard/devices/devices.js:10]: @masterkoppa the new methods should be available now if you'd like to update this pull request.

Comment (masterkoppa) [reply src/controllers/dashboard/devices/devices.js:10]: @dkanada: Thanks for the reminder, I'll find some time soon to update this with that change.

**Comment by masterkoppa** [comment]
The PR has been updated to use the new ApiClient method.

**Review by dkanada — APPROVED** [review]

**Review by ferferga — APPROVED** [review]

**Pull request merged**